play/src/front/Phaser/Game/GameScene.ts

```diff
@@ -925,14 +925,10 @@ export class GameScene extends DirtyScene {
                 });
 
                 this.connection.refreshRoomMessageStream.subscribe((message) => {
-                    if (message.comment) {
-                        refreshPromptStore.set({
-                            comment: message.comment,
-                            timeToRefresh: message.timeToRefresh,
-                        });
-                    } else {
-                        window.location.reload();
-                    }
+                    refreshPromptStore.set({
+                        comment: message.comment,
```

Comment:
Does it make sense to keep the "comment" here?
I don't think there is any path in the code that will fill this "comment" field. Maybe we can remove it altogether?